docs/reference/extensions/django-framework.rst

```diff
@@ -52,6 +49,31 @@ application. In the following example we use it to specify ``libpq-dev``:
          # list required packages or slices for your Django application below.
          - libpq-dev
 
+
+StatsD exporter
+===============
+
+A StatsD exporter is installed alongside the Gunicorn server to export Gunicorn
+server metrics. Three of the `Gunicorn-provided metrics
+<https://docs.gunicorn.org/en/stable/instrumentation.html>`_
+are mapped to new names: ``django_response_code``, ``django_requests`` and
+``django_request_duration``.
```

Comment:
I'd make a table for these.

```suggestion
are mapped to new names:

.. list-table::
    * - StatsD metric
       - Gunicorn metric
    * - ?
       - ``django_response_code``
    * - ``gunicorn.requests``
       - ``django_requests``
    * - ``gunicorn.request.duration``
       - ``django_request_duration``
```

I couldn't see what the response code one maps to.